test/src/websocket-utils.ts

```diff
@@ -0,0 +1,60 @@
+import * as assert from "assert";
+import { sleep } from './test-helpers';
+const WebSocket = require("ws");
```

Comment:
This file is not really "utils". It's really the methods to do debugging communication (over the websocket). Consider "debug-protocol.ts" or "debugger-websocket.ts" -- something along those lines.  Try to avoid naming things "utils" because there are usually more clear names and it encourages better design and organization.